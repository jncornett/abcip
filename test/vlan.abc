d ( stack="eth:vlan:vlan:ip4:tcp" )
a ( syn, 3:cfi=1, pcp=5, vid=0x048; 4:vid=100 )
b ( syn, ack, 3:vid=0x49; 4:vid=101 )
a ( ack, 3:cfi=1, pcp=5 )
b ( ack, len=16 )
