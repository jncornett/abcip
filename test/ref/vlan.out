IP (tos 0x0, ttl 64, id 1, offset 0, flags [none], proto TCP (6), length 40)
    10.1.2.5.48620 > 10.9.8.7.8: Flags [S], cksum 0xb3d7 (correct), seq 1, win 8192, length 0
	0x0000:  0209 0807 0605 0201 0203 0405 8100 b048  ...............H
	0x0010:  8100 0064 0800 4500 0028 0001 0000 4006  ...d..E..(....@.
	0x0020:  5cba 0a01 0205 0a09 0807 bdec 0008 0000  \...............
	0x0030:  0001 0000 0000 5002 2000 b3d7 0000       ......P.......
IP (tos 0x0, ttl 64, id 1, offset 0, flags [none], proto TCP (6), length 40)
    10.9.8.7.8 > 10.1.2.5.48620: Flags [S.], cksum 0xb3c5 (correct), seq 1, ack 2, win 8192, length 0
	0x0000:  0201 0203 0405 0209 0807 0605 8100 0049  ...............I
	0x0010:  8100 0065 0800 4500 0028 0001 0000 4006  ...e..E..(....@.
	0x0020:  5cba 0a09 0807 0a01 0205 0008 bdec 0000  \...............
	0x0030:  0001 0000 0002 5012 2000 b3c5 0000       ......P.......
IP (tos 0x0, ttl 64, id 2, offset 0, flags [none], proto TCP (6), length 40)
    10.1.2.5.48620 > 10.9.8.7.8: Flags [.], cksum 0xb3c6 (correct), ack 1, win 8192, length 0
	0x0000:  0209 0807 0605 0201 0203 0405 8100 b048  ...............H
	0x0010:  8100 0064 0800 4500 0028 0002 0000 4006  ...d..E..(....@.
	0x0020:  5cb9 0a01 0205 0a09 0807 bdec 0008 0000  \...............
	0x0030:  0002 0000 0002 5010 2000 b3c6 0000       ......P.......
IP (tos 0x0, ttl 64, id 2, offset 0, flags [none], proto TCP (6), length 56)
    10.9.8.7.8 > 10.1.2.5.48620: Flags [.], cksum 0x6cdc (correct), seq 1:17, ack 1, win 8192, length 16
	0x0000:  0201 0203 0405 0209 0807 0605 8100 0049  ...............I
	0x0010:  8100 0065 0800 4500 0038 0002 0000 4006  ...e..E..8....@.
	0x0020:  5ca9 0a09 0807 0a01 0205 0008 bdec 0000  \...............
	0x0030:  0002 0000 0002 5010 2000 6cdc 0000 4c6f  ......P...l...Lo
	0x0040:  7265 6d20 6970 7375 6d20 646f 6c6f       rem.ipsum.dolo
